tests/integration/testContainerConfig.ts

```diff
@@ -1,16 +1,24 @@
-import { readFileSync } from 'fs';
 import { container } from 'tsyringe';
 import config from 'config';
-import { MCLogger, IServiceConfig } from '@map-colonies/mc-logger';
+import { Connection, ConnectionOptions, createConnection } from 'typeorm';
+import faker from 'faker';
 import { Services } from '../../src/common/constants';
+import { Entity, IEntity } from '../../src/entity/models/entity';
 
-function registerTestValues(): void {
-  const packageContent = readFileSync('./package.json', 'utf8');
-  const service = JSON.parse(packageContent) as IServiceConfig;
-  const logger = new MCLogger({ log2console: true, level: 'error' }, service);
+const exampleEntity: IEntity = { externalId: faker.random.uuid(), osmId: faker.random.number() };
 
+async function registerTestValues(): Promise<void> {
   container.register(Services.CONFIG, { useValue: config });
-  container.register(Services.LOGGER, { useValue: logger });
+  container.register(Services.LOGGER, { useValue: { log: jest.fn() } });
+
+  const connectionOptions = config.get<ConnectionOptions>('db');
+  const connection = await createConnection({ entities: ['src/entity/models/*.ts'], ...connectionOptions });
+  await connection.synchronize();
+  const repo = connection.getRepository(Entity);
+  const entity = repo.create(exampleEntity);
+  await repo.save(entity);
```

Comment:
I think this does not belong here for 2 reasons:
1. it has nothing to do with test dep injection
2. each test should create its own data. we are hiding some of the test implementations.

Reply:
done